docs/data-sources/blobstore_s3.md

```diff
@@ -0,0 +1,93 @@
+---
+page_title: "Data Source nexus_blobstore_s3"
+subcategory: "Blobstore"
+description: |-
+  Use this to create a Nexus blobstore.
```

Comment:
Description is wrong. Data_resource is not creating a nexus blobstore.

Suggestion:
`Use this to get details of a nexus blobstore s3`